README.md

```diff
@@ -0,0 +1,305 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🙏 Attribution](#attribution)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [JetLogix] <a name="about-project"></a>
+
+JetLogix’s backend is a robust Rails application utilizing PostgreSQL database, managing private jet reservations. It exposes efficient API endpoints to [JetLogix front end](https://github.com/rubydevi/jetlogix-frontend), providing access to the application’s database and  ensuring secure storage and retrieval private jets, user's, and reservation data.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+
+
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+
+
+
+
+  <ul>
+    <li><a href="https://www.mysql.com/">PSQL</a></li>
+  </ul>
+
+
+
+
+  <ul>
+    <li><a href="https://www.mysql.com/">RSpec-Rails</a></li>
+  </ul>
+
+
+
+
+  <ul>
+    <li><a href="https://teamcapybara.github.io/capybara/">Capybara-Rails</a></li>
+  </ul>
+
+
+
+
+  <ul>
+    <li><a href="https://github.com/CanCanCommunity/cancancan">Cancan</a></li>
+  </ul>
+
+
+
+
+  <ul>
+    <li><a href="https://github.com/heartcombo/devise">Devise</a></li>
+  </ul>
+
+
+  <ul>
+    <li><a href="https://github.com/ged/ruby-pg">Ruby Pg</a></li>
+  </ul>
+
+
+<!--
+<summary>Rubocop</summary>
+  <ul>
+    <li><a href="https://github.com/rubocop/rubocop-rails">Rubocop</a></li>
+  </ul>
+
+
+
+<summary>Selenium Web driver</summary>
+  <ul>
+    <li><a href="https://rubygems.org/gems/selenium-webdriver/versions/4.4.0">Selenium Web driver</a></li>
+  </ul> -->
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Exposes robust API endpoints to manage and access the application’s rich database.
+
+> Controllers to manage jets and reservations, allowing CRUD (Create, Read, Update, Delete) operations.
+
+> Secure user authentication and profile management
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Live Demo of this application:
+
+- [Live Demo Link](https:// )
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Presentation Demo <a name="video-presentation"></a>
+
+> Video Presentation of this application:
+
+- [Video Presentation]( )
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+cd my-folder
+git clone https://github.com/rubydevi/jetlogix-backend.git
+```
+
+
+### Install
+
+Install this project with:
+
+    ```sh
+    cd my-project
+    bundle install
+    ```sh
```

Comment:
please kindly fix the installation section. every ting look cool nice work
![image](https://github.com/rubydevi/jetlogix-backend/assets/99332080/39f847e4-d18b-4730-b043-4af5279a6d92)

Reply:
Thanks 👍 I'll definitely do that.